app/models/fraud/indicators/routing_number.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: fraud_indicators_routing_numbers
+#
+#  id             :bigint           not null, primary key
+#  activated_at   :datetime
+#  bank_name      :string
+#  routing_number :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#
+module Fraud
```

Comment:
Not a blocker, but given that the dual write stuff is subtle, I would prefer to split that into its own PR, get it deployed, remove the ugly dual write stuff, _then_ add the new Fraud::Indicators.